voter-dapp/src/ActiveRequests.js

```diff
@@ -164,6 +194,7 @@ function ActiveRequests() {
         <TableHead>
           <TableRow>
             <TableCell>Price Feed</TableCell>
+            <TableCell>?</TableCell>
```

Comment:
What is this?

Reply:
Replaced it with a checkbox that's always disabled.

Reply:
SGTM.